### PR TITLE
Fix the REPL application

### PR DESCRIPTION
**Pull request by jwerle**
This pull request fixes the REPL application and improves the experience. The REPL stopped working during the runtime refactor

### CODE REVIEW

File: bin/repl.js [hunk 1]
```diff
@@ -151,53 +168,60 @@ async function onmessage (message) {
...
     server.setupHistory(HISTORY_PATH, (err) => {
       if (err) {
         console.warn(err.message || err)
       }
     })
 
     server.on('exit', () => {
-      conn.write('ipc://exit?index=0\n')
-      setTimeout(() => conn.destroy(), 32)
+      connection.write('ipc://send?event=exit&index=0&value={}\n')
```
Comment (chicoxyzzy) [inline bin/repl.js]: I think value should be an exit code?

File: console.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { format } from './util.js'
+
+const console = typeof window !== 'undefined' ? window.console ?? {} : {}
```
Comment (chicoxyzzy) [inline console.js:3]:
```suggestion
const console = window?.console ?? {}
```

File: repl/socket.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
-bundle_identifier = co.sockersupply.socket
```
Comment (chicoxyzzy) [inline repl/socket.ini:1]: lol ⚽

Comment (jwerle) [reply repl/socket.ini:1]: 😅

File: console.js (hunk 2 above)
Comment (jwerle) [reply console.js:3]: that results in:

```js
file:///Users/werle/repos/socketsupply/socket-api/console.js:3
const console = window?.console ?? {} : {}
                                      ^

SyntaxError: Unexpected token ':'
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:115:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:289:14)
```

when importing it in node, which I do in `bin/repl.js`

Comment (chicoxyzzy) [reply console.js:3]: `: {}` is unnecessary there

**Review by chicoxyzzy — APPROVED** [review]

**Review by heapwolf — APPROVED** [review]

**Pull request merged**